src/components/Login.js

```diff
@@ -0,0 +1,40 @@
+import React, {useState, useEffect} from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css'
+import './component.css'
+
+
+const Login = props =>{
+const {name, link} = props
+return (
+   <div>
+      <div className="card col-sm-4">
+          <div className="card__container">
+              <div>
+                <h1 className="card__title">Login</h1>
+              </div>
+              
+              <div className="row">
+                <input type="text" className="add__input" placeholder="Username" />
```

Comment:
you can use reactstrap elements. https://reactstrap.github.io/

Reply:
Done